bim_webviewer/bim_web_viewer.md

```diff
@@ -444,6 +444,34 @@ Camera
 
 ---
 
+### Get Snapshot Data URL
+
+<p class="heading-link-container"><a class="heading-link" href="#get-snapshot-data-url"></a></p>
+
+```js
+getSnapshotDataUrl(params);
+```
+
+#### Description
+
+Returns a data url of an image produced by the current render view.
+
+#### Parameters
+
+| Field Name | Required | Type | Description |
+| - | - | - | - |
+| params | false | Object | A JavaScript object that can have the following keys: 'color': Background css color, can use hex value ('#00ff00'), or color labels ('red') |
```

Comment:
Hi @sbieserprocore , would we pass in `height` and `width` in the `params` object?